out_command.go

```diff
@@ -46,6 +46,10 @@ func (c *OutCommand) Run(sourceDir string, request OutRequest) (OutResponse, err
 		Description: github.String(request.Params.Description),
 	}
 
+	if request.Params.Target != "" {
+	    newStatus.EnvironmentURL = github.String(request.Params.Target)
```

Comment:
This seems to conflate target and environment? Should the `Params` field also be `EnvironmentURL`?

Reply:
sure, i was originally doing this for TargetUrl and then they did the old api update on me

Reply:
@eliotstocker Ping. :)